save-demo/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 server.port=5401
 management.endpoints.web.exposure.include=health,info,prometheus
 spring.codec.max-in-memory-size=100MB
+demo.fileStorage.location=${user.home}/.save-cloud/cnb/files
```

Comment:
what's about prod configuration?

Reply:
yeap, but I'm asking more about using `${user.home}` in prod

Reply:
sandbox and backend use `/home/cnb/files` as file storage location. better to use the similar location

Reply:
Will do it as a part of #1450

Reply:
Done